pipeline_steps/common.groovy

```diff
@@ -1625,35 +1625,44 @@ void createComponentGateTrigger(String name, String repoUrl, String jiraProjectK
   }
 }
 
+List getMatchingJobs(String regexMatch){
```

Comment:
Note that I've not implemented the use of this in ``pipeline_steps/gate.groovy`` on purpose. The implementation there is more complex and using this function is not really the focus of this card's work. I've registered https://rpc-openstack.atlassian.net/browse/RE-1647 to look at how we can consolidate the approaches if it makes sense to do so.